pygmt/helpers/decorators.py

```diff
@@ -163,6 +163,9 @@
             (:gmt-term:`PROJ_MEAN_RADIUS`), and the specification of latitude type
             (:gmt-term:`PROJ_AUX_LATITUDE`). Geodesic distance calculations is also
             controlled by method (:gmt-term:`PROJ_GEODESIC`).""",
+    "l": r"""
+        label : str
+        Add a legend entry for the symbol or line being plotted.""",
```

Comment:
It's not correctly rendered:
https://pygmt-fidc7hjbt-gmt.vercel.app/api/generated/pygmt.Figure.plot.html#pygmt.Figure.plot

![image](https://user-images.githubusercontent.com/3974108/128097706-51531984-368b-4869-9c41-7c12f4629bf7.png)

Perhaps need 4 more whitespaces here.
```suggestion
            Add a legend entry for the symbol or line being plotted.""",
```